.github/ISSUE_TEMPLATE

```diff
@@ -0,0 +1,29 @@
+#### Is this a Bug or Feature request?:
+
+
+#### Steps to reproduce or link to a repro project:
+
+
+#### Expected  behavior:
+
+
+#### Actual behavior:
+
+
+#### Environment Information:
+- **TargetFramework (can be found in .csproj file):**
+
+
+- **Version of `Microsoft.AspNetCore.Mvc` or `Microsoft.AspNetCore.App` or `Microsoft.AspNetCore.All`:**
+
+
+
+<!--
+                            --- NOTE ---
+1.  Try enabling logging (in the most verbose level) and see if the details help
+    you in fixing the issue you are seeing. Share the logs too if it helps in
+    diagnosing the issue faster.
+
+2.  If you are seeing an exception, include the full exceptions details (message
+    and stack trace).
+-->
```

Comment:
This doesn't go far enough, if you don't know about these things you don't know how to configure them. This should link to the docs for the exception page and the configuration for logging.